packages/web/src/pages/audio-rewards-page/components/modals/ChallengeRewardsModal/AudioMatchingRewardsModalContent.tsx

```diff
@@ -72,6 +84,45 @@ const ClaimInProgressSpinner = () => (
   <LoadingSpinner className={styles.spinner} />
 )
 
+/**
+ * Filters for only $AUDIO matching challenges and generates row
+ * items in correct format for SummaryTable. Also calculates the
+ * total claimable amount.
+ */
+const useUndisbursedChallengesCooldown = () => {
+  // Only show the cooldown for the $AUDIO matching challenges.
+  // TODO: PAY-2030 Use cooldown_days from api instead of
+  // isAudioMatchingChallenge and COOLDOWN_DAYS.
+  const challenges = useSelector(getUndisbursedUserChallenges)
+    .filter((c) => isAudioMatchingChallenge(c.challenge_id))
+    .map((c) => ({ ...c, created_at: moment(c.created_at) }))
+  const now = moment.utc()
```

Comment:
we opted to start using dayjs instead of moment a bit ago btw (it's smaller) and we should eventually be able to drop moment